drafts/draft-offchain-message-signing.md

```diff
@@ -0,0 +1,129 @@
+---
+tzip: tbd
+title: Off-Chain Message Signing
+status: Draft
+author: Klas Harrysson <klas@kukai.app>, Carlo van Driesten <carlo.van-driesten@vdl.digital>
+type: I
+created: 2024-02-21
+date: To be determined
+requires: ["draft-signer-requests"]
+---
+
+## Abstract
+
+There is no formal message signing standard for Tezos yet. Message signing has many use cases. It can for example be used for code signing, so that authenticity and integrity can be validated. Right now decentralized applications (dApps) and wallets are relying on an informal way of doing message signing by using Michelson data (magic byte `0x05`), something it was never intended for. In addition the [failing_noop (tag 17)][] was not adopted for the use of off-chain messages due to the lack of documentation. This TZIP aims to solve that problem and defines a standard that is simple, secure, future-proof and compatible with hardware wallets.
```

Comment:
```suggestion
There is no formal message signing standard for Tezos yet. Message signing has many use cases. It can for example be used for code signing, so that authenticity and integrity can be validated. Right now decentralized applications (dApps) and wallets are relying on an informal way of doing message signing by using Michelson data (magic byte `0x05`), something it was never intended for. In addition the [failing_noop (tag 17)][] was not adopted for the use of off-chain messages due to the lack of documentation. This TZIP aims to solve that problem and defines a standard that is simple, secure, extendable and compatible with hardware wallets.
```
remove "future-proofed" since we decided to remove versioning. Extendable thanks to the interface and backwards compatability